src/slangpy_ext/device/device.cpp

```diff
@@ -128,6 +129,41 @@ SGL_PY_EXPORT(device_device)
         .def_ro("luid", &AdapterInfo::luid, D(AdapterInfo, luid))
         .def("__repr__", &AdapterInfo::to_string);
 
+    nb::class_<BindlessDesc>(m, "BindlessDesc", D_NA(BindlessDesc))
+        .def_rw("buffer_count", &BindlessDesc::buffer_count, D_NA(BindlessDesc, buffer_count))
+        .def_rw("texture_count", &BindlessDesc::texture_count, D_NA(BindlessDesc, texture_count))
+        .def_rw("sampler_count", &BindlessDesc::sampler_count, D_NA(BindlessDesc, sampler_count))
+        .def_rw(
+            "acceleration_structure_count",
+            &BindlessDesc::acceleration_structure_count,
+            D_NA(BindlessDesc, acceleration_structure_count)
+        )
+        .def(
```

Comment:
instead of this we could do the implicit conversion from a dict as we do with other descs